source/robot_model/src/Model.cpp

```diff
@@ -347,56 +347,54 @@ void Model::print_qp_problem() {
   }
 }
 
+bool Model::in_range(const Eigen::VectorXd& vector,
+                     const Eigen::VectorXd& lower_limits,
+                     const Eigen::VectorXd& upper_limits) {
+  return ((vector.array() >= lower_limits.array()).all() && (vector.array() <= upper_limits.array()).all());
+}
+
 bool Model::in_range(const state_representation::JointPositions& joint_positions) {
-  return this->in_range(joint_positions.get_positions(), this->robot_model_.lowerPositionLimit,
-                                                         this->robot_model_.upperPositionLimit);
+  return this->in_range(joint_positions.get_positions(),
+                        this->robot_model_.lowerPositionLimit,
+                        this->robot_model_.upperPositionLimit);
 }
 
 bool Model::in_range(const state_representation::JointVelocities& joint_velocities) {
-  return this->in_range(joint_velocities.get_velocities(), -this->robot_model_.velocityLimit,
-                                                            this->robot_model_.velocityLimit);
+  return this->in_range(joint_velocities.get_velocities(),
+                        -this->robot_model_.velocityLimit,
+                        this->robot_model_.velocityLimit);
 }
 
 bool Model::in_range(const state_representation::JointTorques& joint_torques) {
-  return this->in_range(joint_torques.get_torques(), -this->robot_model_.effortLimit,
-                                                      this->robot_model_.effortLimit);
+  return this->in_range(joint_torques.get_torques(),
+                        -this->robot_model_.effortLimit,
+                        this->robot_model_.effortLimit);
 }
 
 bool Model::in_range(const state_representation::JointState& joint_state) {
-  return (this->in_range(joint_state.get_positions(), this->robot_model_.lowerPositionLimit,
-                                                       this->robot_model_.upperPositionLimit)
-       && this->in_range(joint_state.get_velocities(), -this->robot_model_.velocityLimit,
-                                                         this->robot_model_.velocityLimit)
-       && this->in_range(joint_state.get_torques(), -this->robot_model_.effortLimit,
-                                                       this->robot_model_.effortLimit));
+  using namespace state_representation;
+  return (this->in_range(static_cast<JointPositions>(joint_state))
+      && this->in_range(static_cast<JointVelocities>(joint_state))
+      && this->in_range(static_cast<JointTorques>(joint_state)));
```

Comment:
Is the scoped namespace really preferable to being explicit here? It doesn't take up that much more space and would be more consistent with the rest of the source file. 
```suggestion
  return (this->in_range(static_cast<state_representation::JointPositions>(joint_state))
      && this->in_range(static_cast<state_representation::JointVelocities>(joint_state))
      && this->in_range(static_cast<state_representation::JointTorques>(joint_state)));
```

Reply:
Nah I think you are right this is better